emcc.py

```diff
@@ -1025,17 +1025,6 @@ def check(input_file):
 
     # Use settings
 
-    try:
-      assert shared.Settings.ASM_JS > 0, 'ASM_JS must be enabled in fastcomp'
```

Comment:
Can we remove the direct entries for these from Settings.js (perhaps also assert that a setting cannot be both at the top level and also in REMOVED_SETTINGS?)

Reply:
Nice!